cluster-scope/base/operators.coreos.com/subscriptions/vertical-pod-autoscaler/subscription.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: vertical-pod-autoscaler
+spec:
+  channel: '4.7'
```

Comment:
See @tumido's comments on https://github.com/operate-first/apps/pull/1175. Specifically:

- You need to update the `channel`
- You need to ensure the `openshift-vertical-pod-autoscaler` namespace exists